BusinessLogicLayer/Address.java

```diff
@@ -72,4 +77,20 @@ public Address(Integer AddressID, Integer UnitNo, String Street, String Suburb,
         this.City = City;
         this.PhoneNo = PhoneNo;
     }
+
+    public List<Address> addresses( ) throws FileNotFoundException{
+        DataHandler dh = new DataHandler();
+        List<Address> aList = dh.GetAddress();
+        if(!aList.isEmpty()){
+            return aList;
+        }
+        else{
+            return null;
+        }
+    }
+
+    public String ToString()
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.